packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleImageViewer/SingleImageViewer.stories.js

```diff
@@ -42,10 +33,9 @@ const mockStore = {
   drawing: {
     addToStream: sinon.stub()
   },
-  subjectViewer: {
-    enableRotation: () => null,
-    setOnZoom: setZoomCallback
-  },
+  subjectViewer: SubjectViewerStore.create({
+    move: true,
```

Comment:
I don't think all the stories should be demoing the same pan/zoom state? It makes it seem like the subject viewer is always in this mode.

Reply:
I don’t think they are. `move` is false for the first two.

Reply:
I’m confused by this code too. `move` should not be true for all three stories.

Reply:
I think setting `move: true` might be right. `move` is always true in the actual subject viewer, unless we're using the mouse to draw on, not drag, the image.

Reply:
I've expanded the tools in the story to include the toggle between annotating and panning the subject. With the default `subjectViewer` model, annotation is enabled but PFE has movement enabled by default unless the current task is a drawing task.